src/core/ConfigManager.cpp

```diff
@@ -81,6 +82,36 @@ ConfigManager::~ConfigManager()
 
 
 
+void ConfigManager::upgrade()
+{
+	// Skip the upgrade if versions match
+	if ( m_version == LMMS_VERSION )
+	{
+		return;
+	}
+
+	ProjectVersion createdWith = m_version;
+	
+	// Remove trailing " (bad latency!)" string which was once saved with PulseAudio
```

Comment:
This is a really nice addition to try to prevent and track backwards compatibility issues.